sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/implementation/AuthorizationProperties.java

```diff
@@ -9,11 +9,21 @@
 
     private List<String> scopes;
 
+    private Boolean onDemand = false;
+
```

Comment:
Can we use the following code?
```
    private boolean onDemand = false;

    public boolean isOnDemand() {
        return onDemand;
    }

    public void setOnDemand(boolean onDemand) {
        this.onDemand = onDemand;
    }
```

1. Use `boolean` instead of `Boolean`. 'Boxing' is not necessary.
2, `isOnDemand` is easier to understand than `getOnDemand`